src/Sulu/Bundle/MediaBundle/Entity/MediaRepository.php

```diff
@@ -227,6 +227,6 @@ public function findSupplierMedia($collectionId, $limit, $offset)
 
         $paginator = new Paginator($query);
 
-        return ['media'=>$paginator, 'count'=>$count];
+        return ['media' => $paginator, 'count' => $count];
```

Comment:
Not related to this PR, but our coding standards use `array` not the short `[ ]` syntax.